2023-09-14 11:08:14,782:WARNING:
'cuml' is a soft dependency and not included in the pycaret installation. Please run: `pip install cuml` to install.
2023-09-14 11:08:14,782:WARNING:
'cuml' is a soft dependency and not included in the pycaret installation. Please run: `pip install cuml` to install.
2023-09-14 11:08:14,782:WARNING:
'cuml' is a soft dependency and not included in the pycaret installation. Please run: `pip install cuml` to install.
2023-09-14 11:08:14,782:WARNING:
'cuml' is a soft dependency and not included in the pycaret installation. Please run: `pip install cuml` to install.
2023-09-14 11:08:51,709:INFO:PyCaret ClassificationExperiment
2023-09-14 11:08:51,709:INFO:Logging name: clf-default-name
2023-09-14 11:08:51,709:INFO:ML Usecase: MLUsecase.CLASSIFICATION
2023-09-14 11:08:51,710:INFO:version 3.1.0
2023-09-14 11:08:51,711:INFO:Initializing setup()
2023-09-14 11:08:51,711:INFO:self.USI: 0633
2023-09-14 11:08:51,711:INFO:self._variable_keys: {'logging_param', 'seed', 'fold_shuffle_param', 'USI', 'is_multiclass', 'y', 'exp_id', 'n_jobs_param', 'memory', 'fold_generator', 'fix_imbalance', '_available_plots', 'idx', 'target_param', 'y_train', 'data', 'X', 'html_param', 'X_train', 'y_test', '_ml_usecase', 'gpu_n_jobs_param', 'fold_groups_param', 'log_plots_param', 'X_test', 'gpu_param', 'exp_name_log', 'pipeline'}
2023-09-14 11:08:51,711:INFO:Checking environment
2023-09-14 11:08:51,711:INFO:python_version: 3.10.10
2023-09-14 11:08:51,711:INFO:python_build: ('tags/v3.10.10:aad5f6a', 'Feb  7 2023 17:20:36')
2023-09-14 11:08:51,711:INFO:machine: AMD64
2023-09-14 11:08:51,711:INFO:platform: Windows-10-10.0.22621-SP0
2023-09-14 11:08:51,712:INFO:Memory: svmem(total=16822788096, available=9719508992, percent=42.2, used=7103279104, free=9719508992)
2023-09-14 11:08:51,713:INFO:Physical Core: 8
2023-09-14 11:08:51,713:INFO:Logical Core: 16
2023-09-14 11:08:51,713:INFO:Checking libraries
2023-09-14 11:08:51,713:INFO:System:
2023-09-14 11:08:51,713:INFO:    python: 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
2023-09-14 11:08:51,713:INFO:executable: d:\BIGAI2023\Python\venv\Scripts\python.exe
2023-09-14 11:08:51,713:INFO:   machine: Windows-10-10.0.22621-SP0
2023-09-14 11:08:51,713:INFO:PyCaret required dependencies:
2023-09-14 11:08:51,752:INFO:                 pip: 23.2.1
2023-09-14 11:08:51,752:INFO:          setuptools: 68.2.0
2023-09-14 11:08:51,752:INFO:             pycaret: 3.1.0
2023-09-14 11:08:51,752:INFO:             IPython: 8.15.0
2023-09-14 11:08:51,752:INFO:          ipywidgets: 8.1.1
2023-09-14 11:08:51,752:INFO:                tqdm: 4.66.1
2023-09-14 11:08:51,752:INFO:               numpy: 1.23.5
2023-09-14 11:08:51,752:INFO:              pandas: 1.5.3
2023-09-14 11:08:51,752:INFO:              jinja2: 3.1.2
2023-09-14 11:08:51,752:INFO:               scipy: 1.10.1
2023-09-14 11:08:51,752:INFO:              joblib: 1.3.2
2023-09-14 11:08:51,752:INFO:             sklearn: 1.2.2
2023-09-14 11:08:51,752:INFO:                pyod: 1.1.0
2023-09-14 11:08:51,752:INFO:            imblearn: 0.11.0
2023-09-14 11:08:51,752:INFO:   category_encoders: 2.6.2
2023-09-14 11:08:51,752:INFO:            lightgbm: 4.1.0
2023-09-14 11:08:51,752:INFO:               numba: 0.57.1
2023-09-14 11:08:51,752:INFO:            requests: 2.31.0
2023-09-14 11:08:51,752:INFO:          matplotlib: 3.7.3
2023-09-14 11:08:51,752:INFO:          scikitplot: 0.3.7
2023-09-14 11:08:51,752:INFO:         yellowbrick: 1.5
2023-09-14 11:08:51,752:INFO:              plotly: 5.16.1
2023-09-14 11:08:51,752:INFO:    plotly-resampler: Not installed
2023-09-14 11:08:51,752:INFO:             kaleido: 0.2.1
2023-09-14 11:08:51,752:INFO:           schemdraw: 0.15
2023-09-14 11:08:51,752:INFO:         statsmodels: 0.14.0
2023-09-14 11:08:51,752:INFO:              sktime: 0.21.1
2023-09-14 11:08:51,752:INFO:               tbats: 1.1.3
2023-09-14 11:08:51,752:INFO:            pmdarima: 2.0.3
2023-09-14 11:08:51,752:INFO:              psutil: 5.9.5
2023-09-14 11:08:51,752:INFO:          markupsafe: 2.1.3
2023-09-14 11:08:51,752:INFO:             pickle5: Not installed
2023-09-14 11:08:51,752:INFO:         cloudpickle: 2.2.1
2023-09-14 11:08:51,752:INFO:         deprecation: 2.1.0
2023-09-14 11:08:51,752:INFO:              xxhash: 3.3.0
2023-09-14 11:08:51,752:INFO:           wurlitzer: Not installed
2023-09-14 11:08:51,752:INFO:PyCaret optional dependencies:
2023-09-14 11:08:51,761:INFO:                shap: Not installed
2023-09-14 11:08:51,761:INFO:           interpret: Not installed
2023-09-14 11:08:51,761:INFO:                umap: Not installed
2023-09-14 11:08:51,761:INFO:     ydata_profiling: Not installed
2023-09-14 11:08:51,761:INFO:  explainerdashboard: Not installed
2023-09-14 11:08:51,761:INFO:             autoviz: Not installed
2023-09-14 11:08:51,761:INFO:           fairlearn: Not installed
2023-09-14 11:08:51,761:INFO:          deepchecks: Not installed
2023-09-14 11:08:51,761:INFO:             xgboost: Not installed
2023-09-14 11:08:51,761:INFO:            catboost: Not installed
2023-09-14 11:08:51,761:INFO:              kmodes: Not installed
2023-09-14 11:08:51,761:INFO:             mlxtend: Not installed
2023-09-14 11:08:51,761:INFO:       statsforecast: Not installed
2023-09-14 11:08:51,761:INFO:        tune_sklearn: Not installed
2023-09-14 11:08:51,761:INFO:                 ray: Not installed
2023-09-14 11:08:51,761:INFO:            hyperopt: Not installed
2023-09-14 11:08:51,761:INFO:              optuna: Not installed
2023-09-14 11:08:51,761:INFO:               skopt: Not installed
2023-09-14 11:08:51,761:INFO:              mlflow: Not installed
2023-09-14 11:08:51,761:INFO:              gradio: Not installed
2023-09-14 11:08:51,761:INFO:             fastapi: Not installed
2023-09-14 11:08:51,761:INFO:             uvicorn: Not installed
2023-09-14 11:08:51,761:INFO:              m2cgen: Not installed
2023-09-14 11:08:51,761:INFO:           evidently: Not installed
2023-09-14 11:08:51,761:INFO:               fugue: Not installed
2023-09-14 11:08:51,761:INFO:           streamlit: Not installed
2023-09-14 11:08:51,761:INFO:             prophet: Not installed
2023-09-14 11:08:51,761:INFO:None
2023-09-14 11:08:51,761:INFO:Set up data.
2023-09-14 11:08:51,767:INFO:Set up folding strategy.
2023-09-14 11:08:51,767:INFO:Set up train/test split.
2023-09-14 11:08:51,770:INFO:Set up index.
2023-09-14 11:08:51,770:INFO:Assigning column types.
2023-09-14 11:08:51,773:INFO:Engine successfully changes for model 'lr' to 'sklearn'.
2023-09-14 11:08:51,803:INFO:Engine for model 'knn' has not been set explicitly, hence returning None.
2023-09-14 11:08:51,805:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:08:51,830:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,830:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,866:INFO:Engine for model 'knn' has not been set explicitly, hence returning None.
2023-09-14 11:08:51,866:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:08:51,885:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,885:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,885:INFO:Engine successfully changes for model 'knn' to 'sklearn'.
2023-09-14 11:08:51,914:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:08:51,932:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,932:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,962:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:08:51,981:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,981:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:51,981:INFO:Engine successfully changes for model 'rbfsvm' to 'sklearn'.
2023-09-14 11:08:52,034:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,034:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,084:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,084:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,087:INFO:Preparing preprocessing pipeline...
2023-09-14 11:08:52,088:INFO:Set up simple imputation.
2023-09-14 11:08:52,088:INFO:Set up column name cleaning.
2023-09-14 11:08:52,104:INFO:Finished creating preprocessing pipeline.
2023-09-14 11:08:52,108:INFO:Pipeline: Pipeline(memory=FastMemory(location=C:\Users\asiae\AppData\Local\Temp\joblib),
         steps=[('numerical_imputer',
                 TransformerWrapper(exclude=None,
                                    include=['mean radius', 'mean texture',
                                             'mean perimeter', 'mean area',
                                             'mean smoothness',
                                             'mean compactness',
                                             'mean concavity',
                                             'mean concave points',
                                             'mean symmetry',
                                             'mean fractal dimension',
                                             'radius error', 'texture error',
                                             'perimeter e...
                 TransformerWrapper(exclude=None, include=[],
                                    transformer=SimpleImputer(add_indicator=False,
                                                              copy=True,
                                                              fill_value=None,
                                                              keep_empty_features=False,
                                                              missing_values=nan,
                                                              strategy='most_frequent',
                                                              verbose='deprecated'))),
                ('clean_column_names',
                 TransformerWrapper(exclude=None, include=None,
                                    transformer=CleanColumnNames(match='[\\]\\[\\,\\{\\}\\"\\:]+')))],
         verbose=False)
2023-09-14 11:08:52,108:INFO:Creating final display dataframe.
2023-09-14 11:08:52,152:INFO:Setup _display_container:                     Description             Value
0                    Session id                 0
1                        Target               tgt
2                   Target type            Binary
3           Original data shape         (569, 31)
4        Transformed data shape         (569, 31)
5   Transformed train set shape         (455, 31)
6    Transformed test set shape         (114, 31)
7              Numeric features                30
8                    Preprocess              True
9               Imputation type            simple
10           Numeric imputation              mean
11       Categorical imputation              mode
12               Fold Generator   StratifiedKFold
13                  Fold Number                10
14                     CPU Jobs                -1
15                      Use GPU             False
16               Log Experiment             False
17              Experiment Name  clf-default-name
18                          USI              0633
2023-09-14 11:08:52,217:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,217:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,269:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,269:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:08:52,270:INFO:setup() successfully completed in 0.56s...............
2023-09-14 11:09:35,715:INFO:PyCaret ClassificationExperiment
2023-09-14 11:09:35,715:INFO:Logging name: clf-default-name
2023-09-14 11:09:35,715:INFO:ML Usecase: MLUsecase.CLASSIFICATION
2023-09-14 11:09:35,715:INFO:version 3.1.0
2023-09-14 11:09:35,715:INFO:Initializing setup()
2023-09-14 11:09:35,715:INFO:self.USI: 0ddc
2023-09-14 11:09:35,715:INFO:self._variable_keys: {'logging_param', 'seed', 'fold_shuffle_param', 'USI', 'is_multiclass', 'y', 'exp_id', 'n_jobs_param', 'memory', 'fold_generator', 'fix_imbalance', '_available_plots', 'idx', 'target_param', 'y_train', 'data', 'X', 'html_param', 'X_train', 'y_test', '_ml_usecase', 'gpu_n_jobs_param', 'fold_groups_param', 'log_plots_param', 'X_test', 'gpu_param', 'exp_name_log', 'pipeline'}
2023-09-14 11:09:35,715:INFO:Checking environment
2023-09-14 11:09:35,715:INFO:python_version: 3.10.10
2023-09-14 11:09:35,715:INFO:python_build: ('tags/v3.10.10:aad5f6a', 'Feb  7 2023 17:20:36')
2023-09-14 11:09:35,715:INFO:machine: AMD64
2023-09-14 11:09:35,715:INFO:platform: Windows-10-10.0.22621-SP0
2023-09-14 11:09:35,717:INFO:Memory: svmem(total=16822788096, available=9681641472, percent=42.4, used=7141146624, free=9681641472)
2023-09-14 11:09:35,717:INFO:Physical Core: 8
2023-09-14 11:09:35,717:INFO:Logical Core: 16
2023-09-14 11:09:35,717:INFO:Checking libraries
2023-09-14 11:09:35,717:INFO:System:
2023-09-14 11:09:35,717:INFO:    python: 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
2023-09-14 11:09:35,717:INFO:executable: d:\BIGAI2023\Python\venv\Scripts\python.exe
2023-09-14 11:09:35,717:INFO:   machine: Windows-10-10.0.22621-SP0
2023-09-14 11:09:35,717:INFO:PyCaret required dependencies:
2023-09-14 11:09:35,717:INFO:                 pip: 23.2.1
2023-09-14 11:09:35,717:INFO:          setuptools: 68.2.0
2023-09-14 11:09:35,717:INFO:             pycaret: 3.1.0
2023-09-14 11:09:35,717:INFO:             IPython: 8.15.0
2023-09-14 11:09:35,717:INFO:          ipywidgets: 8.1.1
2023-09-14 11:09:35,717:INFO:                tqdm: 4.66.1
2023-09-14 11:09:35,717:INFO:               numpy: 1.23.5
2023-09-14 11:09:35,717:INFO:              pandas: 1.5.3
2023-09-14 11:09:35,718:INFO:              jinja2: 3.1.2
2023-09-14 11:09:35,718:INFO:               scipy: 1.10.1
2023-09-14 11:09:35,718:INFO:              joblib: 1.3.2
2023-09-14 11:09:35,718:INFO:             sklearn: 1.2.2
2023-09-14 11:09:35,718:INFO:                pyod: 1.1.0
2023-09-14 11:09:35,718:INFO:            imblearn: 0.11.0
2023-09-14 11:09:35,718:INFO:   category_encoders: 2.6.2
2023-09-14 11:09:35,718:INFO:            lightgbm: 4.1.0
2023-09-14 11:09:35,718:INFO:               numba: 0.57.1
2023-09-14 11:09:35,718:INFO:            requests: 2.31.0
2023-09-14 11:09:35,718:INFO:          matplotlib: 3.7.3
2023-09-14 11:09:35,718:INFO:          scikitplot: 0.3.7
2023-09-14 11:09:35,718:INFO:         yellowbrick: 1.5
2023-09-14 11:09:35,718:INFO:              plotly: 5.16.1
2023-09-14 11:09:35,718:INFO:    plotly-resampler: Not installed
2023-09-14 11:09:35,718:INFO:             kaleido: 0.2.1
2023-09-14 11:09:35,718:INFO:           schemdraw: 0.15
2023-09-14 11:09:35,718:INFO:         statsmodels: 0.14.0
2023-09-14 11:09:35,718:INFO:              sktime: 0.21.1
2023-09-14 11:09:35,718:INFO:               tbats: 1.1.3
2023-09-14 11:09:35,718:INFO:            pmdarima: 2.0.3
2023-09-14 11:09:35,718:INFO:              psutil: 5.9.5
2023-09-14 11:09:35,718:INFO:          markupsafe: 2.1.3
2023-09-14 11:09:35,718:INFO:             pickle5: Not installed
2023-09-14 11:09:35,718:INFO:         cloudpickle: 2.2.1
2023-09-14 11:09:35,718:INFO:         deprecation: 2.1.0
2023-09-14 11:09:35,718:INFO:              xxhash: 3.3.0
2023-09-14 11:09:35,718:INFO:           wurlitzer: Not installed
2023-09-14 11:09:35,718:INFO:PyCaret optional dependencies:
2023-09-14 11:09:35,718:INFO:                shap: Not installed
2023-09-14 11:09:35,718:INFO:           interpret: Not installed
2023-09-14 11:09:35,718:INFO:                umap: Not installed
2023-09-14 11:09:35,718:INFO:     ydata_profiling: Not installed
2023-09-14 11:09:35,718:INFO:  explainerdashboard: Not installed
2023-09-14 11:09:35,718:INFO:             autoviz: Not installed
2023-09-14 11:09:35,719:INFO:           fairlearn: Not installed
2023-09-14 11:09:35,719:INFO:          deepchecks: Not installed
2023-09-14 11:09:35,719:INFO:             xgboost: Not installed
2023-09-14 11:09:35,719:INFO:            catboost: Not installed
2023-09-14 11:09:35,719:INFO:              kmodes: Not installed
2023-09-14 11:09:35,719:INFO:             mlxtend: Not installed
2023-09-14 11:09:35,719:INFO:       statsforecast: Not installed
2023-09-14 11:09:35,719:INFO:        tune_sklearn: Not installed
2023-09-14 11:09:35,719:INFO:                 ray: Not installed
2023-09-14 11:09:35,719:INFO:            hyperopt: Not installed
2023-09-14 11:09:35,719:INFO:              optuna: Not installed
2023-09-14 11:09:35,719:INFO:               skopt: Not installed
2023-09-14 11:09:35,719:INFO:              mlflow: Not installed
2023-09-14 11:09:35,719:INFO:              gradio: Not installed
2023-09-14 11:09:35,719:INFO:             fastapi: Not installed
2023-09-14 11:09:35,719:INFO:             uvicorn: Not installed
2023-09-14 11:09:35,719:INFO:              m2cgen: Not installed
2023-09-14 11:09:35,719:INFO:           evidently: Not installed
2023-09-14 11:09:35,719:INFO:               fugue: Not installed
2023-09-14 11:09:35,719:INFO:           streamlit: Not installed
2023-09-14 11:09:35,719:INFO:             prophet: Not installed
2023-09-14 11:09:35,719:INFO:None
2023-09-14 11:09:35,719:INFO:Set up data.
2023-09-14 11:09:35,725:INFO:Set up folding strategy.
2023-09-14 11:09:35,725:INFO:Set up train/test split.
2023-09-14 11:09:35,728:INFO:Set up index.
2023-09-14 11:09:35,728:INFO:Assigning column types.
2023-09-14 11:09:35,730:INFO:Engine successfully changes for model 'lr' to 'sklearn'.
2023-09-14 11:09:35,765:INFO:Engine for model 'knn' has not been set explicitly, hence returning None.
2023-09-14 11:09:35,766:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:09:35,783:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,784:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,818:INFO:Engine for model 'knn' has not been set explicitly, hence returning None.
2023-09-14 11:09:35,818:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:09:35,838:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,838:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,838:INFO:Engine successfully changes for model 'knn' to 'sklearn'.
2023-09-14 11:09:35,868:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:09:35,886:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,886:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,915:INFO:Engine for model 'rbfsvm' has not been set explicitly, hence returning None.
2023-09-14 11:09:35,934:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,934:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,934:INFO:Engine successfully changes for model 'rbfsvm' to 'sklearn'.
2023-09-14 11:09:35,982:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:35,983:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:36,029:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:36,029:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:36,030:INFO:Preparing preprocessing pipeline...
2023-09-14 11:09:36,030:INFO:Set up simple imputation.
2023-09-14 11:09:36,031:INFO:Set up column name cleaning.
2023-09-14 11:09:36,040:INFO:Finished creating preprocessing pipeline.
2023-09-14 11:09:36,043:INFO:Pipeline: Pipeline(memory=FastMemory(location=C:\Users\asiae\AppData\Local\Temp\joblib),
         steps=[('numerical_imputer',
                 TransformerWrapper(exclude=None,
                                    include=['mean radius', 'mean texture',
                                             'mean perimeter', 'mean area',
                                             'mean smoothness',
                                             'mean compactness',
                                             'mean concavity',
                                             'mean concave points',
                                             'mean symmetry',
                                             'mean fractal dimension',
                                             'radius error', 'texture error',
                                             'perimeter e...
                 TransformerWrapper(exclude=None, include=[],
                                    transformer=SimpleImputer(add_indicator=False,
                                                              copy=True,
                                                              fill_value=None,
                                                              keep_empty_features=False,
                                                              missing_values=nan,
                                                              strategy='most_frequent',
                                                              verbose='deprecated'))),
                ('clean_column_names',
                 TransformerWrapper(exclude=None, include=None,
                                    transformer=CleanColumnNames(match='[\\]\\[\\,\\{\\}\\"\\:]+')))],
         verbose=False)
2023-09-14 11:09:36,043:INFO:Creating final display dataframe.
2023-09-14 11:09:36,081:INFO:Setup _display_container:                     Description             Value
0                    Session id                 0
1                        Target               tgt
2                   Target type            Binary
3           Original data shape         (569, 31)
4        Transformed data shape         (569, 31)
5   Transformed train set shape         (455, 31)
6    Transformed test set shape         (114, 31)
7              Numeric features                30
8                    Preprocess              True
9               Imputation type            simple
10           Numeric imputation              mean
11       Categorical imputation              mode
12               Fold Generator   StratifiedKFold
13                  Fold Number                10
14                     CPU Jobs                -1
15                      Use GPU             False
16               Log Experiment             False
17              Experiment Name  clf-default-name
18                          USI              0ddc
2023-09-14 11:09:36,151:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:36,151:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:36,205:WARNING:
'xgboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install xgboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:36,205:WARNING:
'catboost' is a soft dependency and not included in the pycaret installation. Please run: `pip install catboost` to install.
Alternately, you can install this by running `pip install pycaret[models]`
2023-09-14 11:09:36,206:INFO:setup() successfully completed in 0.49s...............
2023-09-14 11:10:48,808:INFO:Initializing compare_models()
2023-09-14 11:10:48,808:INFO:compare_models(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, include=None, fold=None, round=4, cross_validation=True, sort=accuracy, n_select=1, budget_time=None, turbo=True, errors=ignore, fit_kwargs=None, groups=None, experiment_custom_tags=None, probability_threshold=None, verbose=True, parallel=None, caller_params={'self': <pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, 'include': None, 'exclude': None, 'fold': None, 'round': 4, 'cross_validation': True, 'sort': 'accuracy', 'n_select': 1, 'budget_time': None, 'turbo': True, 'errors': 'ignore', 'fit_kwargs': None, 'groups': None, 'experiment_custom_tags': None, 'probability_threshold': None, 'engine': None, 'verbose': True, 'parallel': None, '__class__': <class 'pycaret.classification.oop.ClassificationExperiment'>}, exclude=None)
2023-09-14 11:10:48,808:INFO:Checking exceptions
2023-09-14 11:10:58,023:INFO:Initializing compare_models()
2023-09-14 11:10:58,023:INFO:compare_models(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, include=None, fold=None, round=4, cross_validation=True, sort=Accuracy, n_select=1, budget_time=None, turbo=True, errors=ignore, fit_kwargs=None, groups=None, experiment_custom_tags=None, probability_threshold=None, verbose=True, parallel=None, caller_params={'self': <pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, 'include': None, 'exclude': None, 'fold': None, 'round': 4, 'cross_validation': True, 'sort': 'Accuracy', 'n_select': 1, 'budget_time': None, 'turbo': True, 'errors': 'ignore', 'fit_kwargs': None, 'groups': None, 'experiment_custom_tags': None, 'probability_threshold': None, 'engine': None, 'verbose': True, 'parallel': None, '__class__': <class 'pycaret.classification.oop.ClassificationExperiment'>}, exclude=None)
2023-09-14 11:10:58,023:INFO:Checking exceptions
2023-09-14 11:10:58,026:INFO:Preparing display monitor
2023-09-14 11:10:58,044:INFO:Initializing Logistic Regression
2023-09-14 11:10:58,044:INFO:Total runtime is 0.0 minutes
2023-09-14 11:10:58,046:INFO:SubProcess create_model() called ==================================
2023-09-14 11:10:58,046:INFO:Initializing create_model()
2023-09-14 11:10:58,046:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=lr, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:10:58,046:INFO:Checking exceptions
2023-09-14 11:10:58,046:INFO:Importing libraries
2023-09-14 11:10:58,046:INFO:Copying training dataset
2023-09-14 11:10:58,050:INFO:Defining folds
2023-09-14 11:10:58,050:INFO:Declaring metric variables
2023-09-14 11:10:58,052:INFO:Importing untrained model
2023-09-14 11:10:58,056:INFO:Logistic Regression Imported successfully
2023-09-14 11:10:58,062:INFO:Starting cross validation
2023-09-14 11:10:58,063:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:00,361:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,362:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,369:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,374:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,375:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,379:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,385:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,420:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,442:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,447:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_logistic.py:458: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(

2023-09-14 11:11:00,466:INFO:Calculating mean and std
2023-09-14 11:11:00,467:INFO:Creating metrics dataframe
2023-09-14 11:11:00,471:INFO:Uploading results into container
2023-09-14 11:11:00,471:INFO:Uploading model into container now
2023-09-14 11:11:00,472:INFO:_master_model_container: 1
2023-09-14 11:11:00,472:INFO:_display_container: 2
2023-09-14 11:11:00,472:INFO:LogisticRegression(C=1.0, class_weight=None, dual=False, fit_intercept=True,
                   intercept_scaling=1, l1_ratio=None, max_iter=1000,
                   multi_class='auto', n_jobs=None, penalty='l2',
                   random_state=0, solver='lbfgs', tol=0.0001, verbose=0,
                   warm_start=False)
2023-09-14 11:11:00,472:INFO:create_model() successfully completed......................................
2023-09-14 11:11:00,531:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:00,531:INFO:Creating metrics dataframe
2023-09-14 11:11:00,537:INFO:Initializing K Neighbors Classifier
2023-09-14 11:11:00,537:INFO:Total runtime is 0.041552635033925374 minutes
2023-09-14 11:11:00,547:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:00,547:INFO:Initializing create_model()
2023-09-14 11:11:00,548:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=knn, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:00,548:INFO:Checking exceptions
2023-09-14 11:11:00,548:INFO:Importing libraries
2023-09-14 11:11:00,548:INFO:Copying training dataset
2023-09-14 11:11:00,555:INFO:Defining folds
2023-09-14 11:11:00,555:INFO:Declaring metric variables
2023-09-14 11:11:00,560:INFO:Importing untrained model
2023-09-14 11:11:00,567:INFO:K Neighbors Classifier Imported successfully
2023-09-14 11:11:00,579:INFO:Starting cross validation
2023-09-14 11:11:00,579:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:02,041:INFO:Calculating mean and std
2023-09-14 11:11:02,041:INFO:Creating metrics dataframe
2023-09-14 11:11:02,045:INFO:Uploading results into container
2023-09-14 11:11:02,046:INFO:Uploading model into container now
2023-09-14 11:11:02,046:INFO:_master_model_container: 2
2023-09-14 11:11:02,047:INFO:_display_container: 2
2023-09-14 11:11:02,047:INFO:KNeighborsClassifier(algorithm='auto', leaf_size=30, metric='minkowski',
                     metric_params=None, n_jobs=-1, n_neighbors=5, p=2,
                     weights='uniform')
2023-09-14 11:11:02,047:INFO:create_model() successfully completed......................................
2023-09-14 11:11:02,127:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:02,127:INFO:Creating metrics dataframe
2023-09-14 11:11:02,135:INFO:Initializing Naive Bayes
2023-09-14 11:11:02,135:INFO:Total runtime is 0.06817580064137777 minutes
2023-09-14 11:11:02,137:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:02,137:INFO:Initializing create_model()
2023-09-14 11:11:02,137:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=nb, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:02,137:INFO:Checking exceptions
2023-09-14 11:11:02,137:INFO:Importing libraries
2023-09-14 11:11:02,137:INFO:Copying training dataset
2023-09-14 11:11:02,140:INFO:Defining folds
2023-09-14 11:11:02,140:INFO:Declaring metric variables
2023-09-14 11:11:02,142:INFO:Importing untrained model
2023-09-14 11:11:02,145:INFO:Naive Bayes Imported successfully
2023-09-14 11:11:02,151:INFO:Starting cross validation
2023-09-14 11:11:02,151:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:02,198:INFO:Calculating mean and std
2023-09-14 11:11:02,198:INFO:Creating metrics dataframe
2023-09-14 11:11:02,201:INFO:Uploading results into container
2023-09-14 11:11:02,201:INFO:Uploading model into container now
2023-09-14 11:11:02,201:INFO:_master_model_container: 3
2023-09-14 11:11:02,201:INFO:_display_container: 2
2023-09-14 11:11:02,201:INFO:GaussianNB(priors=None, var_smoothing=1e-09)
2023-09-14 11:11:02,201:INFO:create_model() successfully completed......................................
2023-09-14 11:11:02,254:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:02,254:INFO:Creating metrics dataframe
2023-09-14 11:11:02,262:INFO:Initializing Decision Tree Classifier
2023-09-14 11:11:02,262:INFO:Total runtime is 0.07028758923212687 minutes
2023-09-14 11:11:02,266:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:02,266:INFO:Initializing create_model()
2023-09-14 11:11:02,266:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=dt, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:02,266:INFO:Checking exceptions
2023-09-14 11:11:02,266:INFO:Importing libraries
2023-09-14 11:11:02,266:INFO:Copying training dataset
2023-09-14 11:11:02,268:INFO:Defining folds
2023-09-14 11:11:02,270:INFO:Declaring metric variables
2023-09-14 11:11:02,272:INFO:Importing untrained model
2023-09-14 11:11:02,274:INFO:Decision Tree Classifier Imported successfully
2023-09-14 11:11:02,279:INFO:Starting cross validation
2023-09-14 11:11:02,281:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:02,339:INFO:Calculating mean and std
2023-09-14 11:11:02,339:INFO:Creating metrics dataframe
2023-09-14 11:11:02,341:INFO:Uploading results into container
2023-09-14 11:11:02,341:INFO:Uploading model into container now
2023-09-14 11:11:02,342:INFO:_master_model_container: 4
2023-09-14 11:11:02,342:INFO:_display_container: 2
2023-09-14 11:11:02,342:INFO:DecisionTreeClassifier(ccp_alpha=0.0, class_weight=None, criterion='gini',
                       max_depth=None, max_features=None, max_leaf_nodes=None,
                       min_impurity_decrease=0.0, min_samples_leaf=1,
                       min_samples_split=2, min_weight_fraction_leaf=0.0,
                       random_state=0, splitter='best')
2023-09-14 11:11:02,342:INFO:create_model() successfully completed......................................
2023-09-14 11:11:02,395:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:02,396:INFO:Creating metrics dataframe
2023-09-14 11:11:02,403:INFO:Initializing SVM - Linear Kernel
2023-09-14 11:11:02,403:INFO:Total runtime is 0.07263888915379842 minutes
2023-09-14 11:11:02,407:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:02,407:INFO:Initializing create_model()
2023-09-14 11:11:02,407:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=svm, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:02,407:INFO:Checking exceptions
2023-09-14 11:11:02,407:INFO:Importing libraries
2023-09-14 11:11:02,407:INFO:Copying training dataset
2023-09-14 11:11:02,410:INFO:Defining folds
2023-09-14 11:11:02,410:INFO:Declaring metric variables
2023-09-14 11:11:02,412:INFO:Importing untrained model
2023-09-14 11:11:02,415:INFO:SVM - Linear Kernel Imported successfully
2023-09-14 11:11:02,420:INFO:Starting cross validation
2023-09-14 11:11:02,421:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:02,449:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,449:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,449:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,451:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,453:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,453:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,458:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,459:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,460:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\utils\_available_if.py", line 32, in __get__
    if not self.check(obj):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_stochastic_gradient.py", line 1235, in _check_proba
    raise AttributeError(
AttributeError: probability estimates are not available for loss='hinge'

  warnings.warn(

2023-09-14 11:11:02,478:INFO:Calculating mean and std
2023-09-14 11:11:02,478:INFO:Creating metrics dataframe
2023-09-14 11:11:02,480:INFO:Uploading results into container
2023-09-14 11:11:02,481:INFO:Uploading model into container now
2023-09-14 11:11:02,481:INFO:_master_model_container: 5
2023-09-14 11:11:02,481:INFO:_display_container: 2
2023-09-14 11:11:02,481:INFO:SGDClassifier(alpha=0.0001, average=False, class_weight=None,
              early_stopping=False, epsilon=0.1, eta0=0.001, fit_intercept=True,
              l1_ratio=0.15, learning_rate='optimal', loss='hinge',
              max_iter=1000, n_iter_no_change=5, n_jobs=-1, penalty='l2',
              power_t=0.5, random_state=0, shuffle=True, tol=0.001,
              validation_fraction=0.1, verbose=0, warm_start=False)
2023-09-14 11:11:02,481:INFO:create_model() successfully completed......................................
2023-09-14 11:11:02,534:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:02,534:INFO:Creating metrics dataframe
2023-09-14 11:11:02,540:INFO:Initializing Ridge Classifier
2023-09-14 11:11:02,540:INFO:Total runtime is 0.07493565082550048 minutes
2023-09-14 11:11:02,545:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:02,545:INFO:Initializing create_model()
2023-09-14 11:11:02,545:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ridge, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:02,545:INFO:Checking exceptions
2023-09-14 11:11:02,545:INFO:Importing libraries
2023-09-14 11:11:02,545:INFO:Copying training dataset
2023-09-14 11:11:02,547:INFO:Defining folds
2023-09-14 11:11:02,548:INFO:Declaring metric variables
2023-09-14 11:11:02,551:INFO:Importing untrained model
2023-09-14 11:11:02,553:INFO:Ridge Classifier Imported successfully
2023-09-14 11:11:02,559:INFO:Starting cross validation
2023-09-14 11:11:02,560:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:02,579:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=3.1666e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,581:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=3.73691e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,582:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=3.74962e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,583:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=4.05254e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,583:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=3.01569e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,584:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=4.09064e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,587:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=4.13871e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,587:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,588:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,588:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=3.92802e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,588:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=3.90181e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,588:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,589:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,589:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,591:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\linear_model\_ridge.py:216: LinAlgWarning: Ill-conditioned matrix (rcond=3.21729e-09): result may not be accurate.
  return linalg.solve(A, Xy, assume_a="pos", overwrite_a=True).T

2023-09-14 11:11:02,592:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,594:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,596:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,596:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,600:WARNING:d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py:190: FitFailedWarning: Metric 'make_scorer(roc_auc_score, needs_proba=True, error_score=0.0, average=weighted, multi_class=ovr)' failed and error score 0.0 has been returned instead. If this is a custom metric, this usually means that the error is in the metric code. Full exception below:
Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 76, in _cached_call
    return cache[method]
KeyError: 'predict_proba'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\BIGAI2023\Python\venv\lib\site-packages\pycaret\internal\metrics.py", line 182, in _score
    return super()._score(
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 316, in _score
    y_pred = method_caller(clf, "predict_proba", X)
  File "d:\BIGAI2023\Python\venv\lib\site-packages\sklearn\metrics\_scorer.py", line 78, in _cached_call
    result = getattr(estimator, method)(*args, **kwargs)
AttributeError: 'RidgeClassifier' object has no attribute 'predict_proba'

  warnings.warn(

2023-09-14 11:11:02,619:INFO:Calculating mean and std
2023-09-14 11:11:02,619:INFO:Creating metrics dataframe
2023-09-14 11:11:02,621:INFO:Uploading results into container
2023-09-14 11:11:02,621:INFO:Uploading model into container now
2023-09-14 11:11:02,622:INFO:_master_model_container: 6
2023-09-14 11:11:02,622:INFO:_display_container: 2
2023-09-14 11:11:02,622:INFO:RidgeClassifier(alpha=1.0, class_weight=None, copy_X=True, fit_intercept=True,
                max_iter=None, positive=False, random_state=0, solver='auto',
                tol=0.0001)
2023-09-14 11:11:02,622:INFO:create_model() successfully completed......................................
2023-09-14 11:11:02,674:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:02,674:INFO:Creating metrics dataframe
2023-09-14 11:11:02,681:INFO:Initializing Random Forest Classifier
2023-09-14 11:11:02,681:INFO:Total runtime is 0.07727431853612263 minutes
2023-09-14 11:11:02,685:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:02,686:INFO:Initializing create_model()
2023-09-14 11:11:02,686:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=rf, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:02,686:INFO:Checking exceptions
2023-09-14 11:11:02,686:INFO:Importing libraries
2023-09-14 11:11:02,686:INFO:Copying training dataset
2023-09-14 11:11:02,691:INFO:Defining folds
2023-09-14 11:11:02,691:INFO:Declaring metric variables
2023-09-14 11:11:02,694:INFO:Importing untrained model
2023-09-14 11:11:02,698:INFO:Random Forest Classifier Imported successfully
2023-09-14 11:11:02,703:INFO:Starting cross validation
2023-09-14 11:11:02,704:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:03,025:INFO:Calculating mean and std
2023-09-14 11:11:03,025:INFO:Creating metrics dataframe
2023-09-14 11:11:03,028:INFO:Uploading results into container
2023-09-14 11:11:03,028:INFO:Uploading model into container now
2023-09-14 11:11:03,028:INFO:_master_model_container: 7
2023-09-14 11:11:03,029:INFO:_display_container: 2
2023-09-14 11:11:03,029:INFO:RandomForestClassifier(bootstrap=True, ccp_alpha=0.0, class_weight=None,
                       criterion='gini', max_depth=None, max_features='sqrt',
                       max_leaf_nodes=None, max_samples=None,
                       min_impurity_decrease=0.0, min_samples_leaf=1,
                       min_samples_split=2, min_weight_fraction_leaf=0.0,
                       n_estimators=100, n_jobs=-1, oob_score=False,
                       random_state=0, verbose=0, warm_start=False)
2023-09-14 11:11:03,029:INFO:create_model() successfully completed......................................
2023-09-14 11:11:03,088:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:03,088:INFO:Creating metrics dataframe
2023-09-14 11:11:03,096:INFO:Initializing Quadratic Discriminant Analysis
2023-09-14 11:11:03,096:INFO:Total runtime is 0.08420284191767373 minutes
2023-09-14 11:11:03,099:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:03,100:INFO:Initializing create_model()
2023-09-14 11:11:03,100:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=qda, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:03,100:INFO:Checking exceptions
2023-09-14 11:11:03,100:INFO:Importing libraries
2023-09-14 11:11:03,100:INFO:Copying training dataset
2023-09-14 11:11:03,102:INFO:Defining folds
2023-09-14 11:11:03,102:INFO:Declaring metric variables
2023-09-14 11:11:03,105:INFO:Importing untrained model
2023-09-14 11:11:03,109:INFO:Quadratic Discriminant Analysis Imported successfully
2023-09-14 11:11:03,114:INFO:Starting cross validation
2023-09-14 11:11:03,114:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:03,165:INFO:Calculating mean and std
2023-09-14 11:11:03,166:INFO:Creating metrics dataframe
2023-09-14 11:11:03,168:INFO:Uploading results into container
2023-09-14 11:11:03,168:INFO:Uploading model into container now
2023-09-14 11:11:03,168:INFO:_master_model_container: 8
2023-09-14 11:11:03,168:INFO:_display_container: 2
2023-09-14 11:11:03,168:INFO:QuadraticDiscriminantAnalysis(priors=None, reg_param=0.0,
                              store_covariance=False, tol=0.0001)
2023-09-14 11:11:03,168:INFO:create_model() successfully completed......................................
2023-09-14 11:11:03,221:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:03,221:INFO:Creating metrics dataframe
2023-09-14 11:11:03,229:INFO:Initializing Ada Boost Classifier
2023-09-14 11:11:03,229:INFO:Total runtime is 0.0864147106806437 minutes
2023-09-14 11:11:03,233:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:03,233:INFO:Initializing create_model()
2023-09-14 11:11:03,233:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ada, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:03,233:INFO:Checking exceptions
2023-09-14 11:11:03,233:INFO:Importing libraries
2023-09-14 11:11:03,233:INFO:Copying training dataset
2023-09-14 11:11:03,236:INFO:Defining folds
2023-09-14 11:11:03,236:INFO:Declaring metric variables
2023-09-14 11:11:03,238:INFO:Importing untrained model
2023-09-14 11:11:03,240:INFO:Ada Boost Classifier Imported successfully
2023-09-14 11:11:03,246:INFO:Starting cross validation
2023-09-14 11:11:03,246:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:03,478:INFO:Calculating mean and std
2023-09-14 11:11:03,479:INFO:Creating metrics dataframe
2023-09-14 11:11:03,481:INFO:Uploading results into container
2023-09-14 11:11:03,481:INFO:Uploading model into container now
2023-09-14 11:11:03,481:INFO:_master_model_container: 9
2023-09-14 11:11:03,481:INFO:_display_container: 2
2023-09-14 11:11:03,481:INFO:AdaBoostClassifier(algorithm='SAMME.R', base_estimator='deprecated',
                   estimator=None, learning_rate=1.0, n_estimators=50,
                   random_state=0)
2023-09-14 11:11:03,481:INFO:create_model() successfully completed......................................
2023-09-14 11:11:03,543:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:03,544:INFO:Creating metrics dataframe
2023-09-14 11:11:03,552:INFO:Initializing Gradient Boosting Classifier
2023-09-14 11:11:03,552:INFO:Total runtime is 0.09178793430328368 minutes
2023-09-14 11:11:03,554:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:03,554:INFO:Initializing create_model()
2023-09-14 11:11:03,555:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=gbc, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:03,555:INFO:Checking exceptions
2023-09-14 11:11:03,555:INFO:Importing libraries
2023-09-14 11:11:03,555:INFO:Copying training dataset
2023-09-14 11:11:03,557:INFO:Defining folds
2023-09-14 11:11:03,557:INFO:Declaring metric variables
2023-09-14 11:11:03,559:INFO:Importing untrained model
2023-09-14 11:11:03,561:INFO:Gradient Boosting Classifier Imported successfully
2023-09-14 11:11:03,566:INFO:Starting cross validation
2023-09-14 11:11:03,566:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:04,155:INFO:Calculating mean and std
2023-09-14 11:11:04,156:INFO:Creating metrics dataframe
2023-09-14 11:11:04,158:INFO:Uploading results into container
2023-09-14 11:11:04,158:INFO:Uploading model into container now
2023-09-14 11:11:04,159:INFO:_master_model_container: 10
2023-09-14 11:11:04,159:INFO:_display_container: 2
2023-09-14 11:11:04,159:INFO:GradientBoostingClassifier(ccp_alpha=0.0, criterion='friedman_mse', init=None,
                           learning_rate=0.1, loss='log_loss', max_depth=3,
                           max_features=None, max_leaf_nodes=None,
                           min_impurity_decrease=0.0, min_samples_leaf=1,
                           min_samples_split=2, min_weight_fraction_leaf=0.0,
                           n_estimators=100, n_iter_no_change=None,
                           random_state=0, subsample=1.0, tol=0.0001,
                           validation_fraction=0.1, verbose=0,
                           warm_start=False)
2023-09-14 11:11:04,159:INFO:create_model() successfully completed......................................
2023-09-14 11:11:04,219:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:04,219:INFO:Creating metrics dataframe
2023-09-14 11:11:04,227:INFO:Initializing Linear Discriminant Analysis
2023-09-14 11:11:04,227:INFO:Total runtime is 0.10305038690567016 minutes
2023-09-14 11:11:04,231:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:04,232:INFO:Initializing create_model()
2023-09-14 11:11:04,232:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=lda, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:04,232:INFO:Checking exceptions
2023-09-14 11:11:04,232:INFO:Importing libraries
2023-09-14 11:11:04,232:INFO:Copying training dataset
2023-09-14 11:11:04,234:INFO:Defining folds
2023-09-14 11:11:04,235:INFO:Declaring metric variables
2023-09-14 11:11:04,237:INFO:Importing untrained model
2023-09-14 11:11:04,239:INFO:Linear Discriminant Analysis Imported successfully
2023-09-14 11:11:04,246:INFO:Starting cross validation
2023-09-14 11:11:04,247:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:04,309:INFO:Calculating mean and std
2023-09-14 11:11:04,310:INFO:Creating metrics dataframe
2023-09-14 11:11:04,312:INFO:Uploading results into container
2023-09-14 11:11:04,312:INFO:Uploading model into container now
2023-09-14 11:11:04,312:INFO:_master_model_container: 11
2023-09-14 11:11:04,312:INFO:_display_container: 2
2023-09-14 11:11:04,312:INFO:LinearDiscriminantAnalysis(covariance_estimator=None, n_components=None,
                           priors=None, shrinkage=None, solver='svd',
                           store_covariance=False, tol=0.0001)
2023-09-14 11:11:04,312:INFO:create_model() successfully completed......................................
2023-09-14 11:11:04,363:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:04,363:INFO:Creating metrics dataframe
2023-09-14 11:11:04,372:INFO:Initializing Extra Trees Classifier
2023-09-14 11:11:04,372:INFO:Total runtime is 0.10545576810836792 minutes
2023-09-14 11:11:04,375:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:04,375:INFO:Initializing create_model()
2023-09-14 11:11:04,375:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=et, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:04,375:INFO:Checking exceptions
2023-09-14 11:11:04,375:INFO:Importing libraries
2023-09-14 11:11:04,375:INFO:Copying training dataset
2023-09-14 11:11:04,377:INFO:Defining folds
2023-09-14 11:11:04,377:INFO:Declaring metric variables
2023-09-14 11:11:04,380:INFO:Importing untrained model
2023-09-14 11:11:04,383:INFO:Extra Trees Classifier Imported successfully
2023-09-14 11:11:04,387:INFO:Starting cross validation
2023-09-14 11:11:04,387:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:04,621:INFO:Calculating mean and std
2023-09-14 11:11:04,622:INFO:Creating metrics dataframe
2023-09-14 11:11:04,624:INFO:Uploading results into container
2023-09-14 11:11:04,624:INFO:Uploading model into container now
2023-09-14 11:11:04,625:INFO:_master_model_container: 12
2023-09-14 11:11:04,625:INFO:_display_container: 2
2023-09-14 11:11:04,625:INFO:ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False)
2023-09-14 11:11:04,625:INFO:create_model() successfully completed......................................
2023-09-14 11:11:04,693:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:04,693:INFO:Creating metrics dataframe
2023-09-14 11:11:04,701:INFO:Initializing Light Gradient Boosting Machine
2023-09-14 11:11:04,701:INFO:Total runtime is 0.11095232963562011 minutes
2023-09-14 11:11:04,705:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:04,705:INFO:Initializing create_model()
2023-09-14 11:11:04,706:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=lightgbm, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:04,706:INFO:Checking exceptions
2023-09-14 11:11:04,706:INFO:Importing libraries
2023-09-14 11:11:04,706:INFO:Copying training dataset
2023-09-14 11:11:04,709:INFO:Defining folds
2023-09-14 11:11:04,709:INFO:Declaring metric variables
2023-09-14 11:11:04,712:INFO:Importing untrained model
2023-09-14 11:11:04,715:INFO:Light Gradient Boosting Machine Imported successfully
2023-09-14 11:11:04,720:INFO:Starting cross validation
2023-09-14 11:11:04,722:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:05,271:INFO:Calculating mean and std
2023-09-14 11:11:05,272:INFO:Creating metrics dataframe
2023-09-14 11:11:05,276:INFO:Uploading results into container
2023-09-14 11:11:05,276:INFO:Uploading model into container now
2023-09-14 11:11:05,277:INFO:_master_model_container: 13
2023-09-14 11:11:05,277:INFO:_display_container: 2
2023-09-14 11:11:05,277:INFO:LGBMClassifier(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
               importance_type='split', learning_rate=0.1, max_depth=-1,
               min_child_samples=20, min_child_weight=0.001, min_split_gain=0.0,
               n_estimators=100, n_jobs=-1, num_leaves=31, objective=None,
               random_state=0, reg_alpha=0.0, reg_lambda=0.0, subsample=1.0,
               subsample_for_bin=200000, subsample_freq=0)
2023-09-14 11:11:05,278:INFO:create_model() successfully completed......................................
2023-09-14 11:11:05,343:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:05,343:INFO:Creating metrics dataframe
2023-09-14 11:11:05,354:INFO:Initializing Dummy Classifier
2023-09-14 11:11:05,354:INFO:Total runtime is 0.12182429234186808 minutes
2023-09-14 11:11:05,357:INFO:SubProcess create_model() called ==================================
2023-09-14 11:11:05,357:INFO:Initializing create_model()
2023-09-14 11:11:05,357:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=dummy, fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=<pycaret.internal.display.display.CommonDisplay object at 0x000002612DA38580>, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:05,357:INFO:Checking exceptions
2023-09-14 11:11:05,357:INFO:Importing libraries
2023-09-14 11:11:05,357:INFO:Copying training dataset
2023-09-14 11:11:05,359:INFO:Defining folds
2023-09-14 11:11:05,359:INFO:Declaring metric variables
2023-09-14 11:11:05,361:INFO:Importing untrained model
2023-09-14 11:11:05,363:INFO:Dummy Classifier Imported successfully
2023-09-14 11:11:05,368:INFO:Starting cross validation
2023-09-14 11:11:05,369:INFO:Cross validating with StratifiedKFold(n_splits=10, random_state=None, shuffle=False), n_jobs=-1
2023-09-14 11:11:05,412:INFO:Calculating mean and std
2023-09-14 11:11:05,413:INFO:Creating metrics dataframe
2023-09-14 11:11:05,415:INFO:Uploading results into container
2023-09-14 11:11:05,415:INFO:Uploading model into container now
2023-09-14 11:11:05,416:INFO:_master_model_container: 14
2023-09-14 11:11:05,416:INFO:_display_container: 2
2023-09-14 11:11:05,416:INFO:DummyClassifier(constant=None, random_state=0, strategy='prior')
2023-09-14 11:11:05,416:INFO:create_model() successfully completed......................................
2023-09-14 11:11:05,470:INFO:SubProcess create_model() end ==================================
2023-09-14 11:11:05,470:INFO:Creating metrics dataframe
2023-09-14 11:11:05,485:INFO:Initializing create_model()
2023-09-14 11:11:05,485:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False), fold=StratifiedKFold(n_splits=10, random_state=None, shuffle=False), round=4, cross_validation=False, predict=False, fit_kwargs={}, groups=None, refit=True, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=True, metrics=None, display=None, model_only=True, return_train_score=False, kwargs={})
2023-09-14 11:11:05,485:INFO:Checking exceptions
2023-09-14 11:11:05,487:INFO:Importing libraries
2023-09-14 11:11:05,487:INFO:Copying training dataset
2023-09-14 11:11:05,491:INFO:Defining folds
2023-09-14 11:11:05,491:INFO:Declaring metric variables
2023-09-14 11:11:05,492:INFO:Importing untrained model
2023-09-14 11:11:05,492:INFO:Declaring custom model
2023-09-14 11:11:05,494:INFO:Extra Trees Classifier Imported successfully
2023-09-14 11:11:05,496:INFO:Cross validation set to False
2023-09-14 11:11:05,496:INFO:Fitting Model
2023-09-14 11:11:05,599:INFO:ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False)
2023-09-14 11:11:05,599:INFO:create_model() successfully completed......................................
2023-09-14 11:11:05,676:INFO:_master_model_container: 14
2023-09-14 11:11:05,676:INFO:_display_container: 2
2023-09-14 11:11:05,676:INFO:ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False)
2023-09-14 11:11:05,676:INFO:compare_models() successfully completed......................................
2023-09-14 11:14:15,475:INFO:Initializing predict_model()
2023-09-14 11:14:15,475:INFO:predict_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False), probability_threshold=None, encoded_labels=False, raw_score=False, round=4, verbose=True, ml_usecase=None, preprocess=True, encode_labels=<function _SupervisedExperiment.predict_model.<locals>.encode_labels at 0x000002612E39A200>)
2023-09-14 11:14:15,475:INFO:Checking exceptions
2023-09-14 11:14:15,475:INFO:Preloading libraries
2023-09-14 11:14:32,944:INFO:Initializing predict_model()
2023-09-14 11:14:32,944:INFO:predict_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False), probability_threshold=None, encoded_labels=False, raw_score=False, round=4, verbose=True, ml_usecase=None, preprocess=True, encode_labels=<function _SupervisedExperiment.predict_model.<locals>.encode_labels at 0x000002612E3989D0>)
2023-09-14 11:14:32,945:INFO:Checking exceptions
2023-09-14 11:14:32,945:INFO:Preloading libraries
2023-09-14 11:16:32,790:INFO:Initializing finalize_model()
2023-09-14 11:16:32,791:INFO:finalize_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False), fit_kwargs=None, groups=None, model_only=False, experiment_custom_tags=None)
2023-09-14 11:16:32,791:INFO:Finalizing ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False)
2023-09-14 11:16:32,795:INFO:Initializing create_model()
2023-09-14 11:16:32,795:INFO:create_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False), fold=None, round=4, cross_validation=False, predict=False, fit_kwargs={}, groups=None, refit=True, probability_threshold=None, experiment_custom_tags=None, verbose=False, system=False, add_to_model_list=False, metrics=None, display=None, model_only=False, return_train_score=False, kwargs={})
2023-09-14 11:16:32,795:INFO:Checking exceptions
2023-09-14 11:16:32,797:INFO:Importing libraries
2023-09-14 11:16:32,798:INFO:Copying training dataset
2023-09-14 11:16:32,798:INFO:Defining folds
2023-09-14 11:16:32,798:INFO:Declaring metric variables
2023-09-14 11:16:32,798:INFO:Importing untrained model
2023-09-14 11:16:32,798:INFO:Declaring custom model
2023-09-14 11:16:32,798:INFO:Extra Trees Classifier Imported successfully
2023-09-14 11:16:32,799:INFO:Cross validation set to False
2023-09-14 11:16:32,799:INFO:Fitting Model
2023-09-14 11:16:32,903:INFO:Pipeline(memory=Memory(location=None),
         steps=[('numerical_imputer',
                 TransformerWrapper(exclude=None,
                                    include=['mean radius', 'mean texture',
                                             'mean perimeter', 'mean area',
                                             'mean smoothness',
                                             'mean compactness',
                                             'mean concavity',
                                             'mean concave points',
                                             'mean symmetry',
                                             'mean fractal dimension',
                                             'radius error', 'texture error',
                                             'perimeter error', 'area error',
                                             'smoothness error',
                                             'com...
                 ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0,
                                      class_weight=None, criterion='gini',
                                      max_depth=None, max_features='sqrt',
                                      max_leaf_nodes=None, max_samples=None,
                                      min_impurity_decrease=0.0,
                                      min_samples_leaf=1, min_samples_split=2,
                                      min_weight_fraction_leaf=0.0,
                                      n_estimators=100, n_jobs=-1,
                                      oob_score=False, random_state=0,
                                      verbose=0, warm_start=False))],
         verbose=False)
2023-09-14 11:16:32,903:INFO:create_model() successfully completed......................................
2023-09-14 11:16:32,955:INFO:_master_model_container: 14
2023-09-14 11:16:32,956:INFO:_display_container: 4
2023-09-14 11:16:32,959:INFO:Pipeline(memory=Memory(location=None),
         steps=[('numerical_imputer',
                 TransformerWrapper(exclude=None,
                                    include=['mean radius', 'mean texture',
                                             'mean perimeter', 'mean area',
                                             'mean smoothness',
                                             'mean compactness',
                                             'mean concavity',
                                             'mean concave points',
                                             'mean symmetry',
                                             'mean fractal dimension',
                                             'radius error', 'texture error',
                                             'perimeter error', 'area error',
                                             'smoothness error',
                                             'com...
                 ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0,
                                      class_weight=None, criterion='gini',
                                      max_depth=None, max_features='sqrt',
                                      max_leaf_nodes=None, max_samples=None,
                                      min_impurity_decrease=0.0,
                                      min_samples_leaf=1, min_samples_split=2,
                                      min_weight_fraction_leaf=0.0,
                                      n_estimators=100, n_jobs=-1,
                                      oob_score=False, random_state=0,
                                      verbose=0, warm_start=False))],
         verbose=False)
2023-09-14 11:16:32,959:INFO:finalize_model() successfully completed......................................
2023-09-14 11:18:36,687:INFO:Initializing predict_model()
2023-09-14 11:18:36,687:INFO:predict_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False), probability_threshold=None, encoded_labels=False, raw_score=False, round=4, verbose=True, ml_usecase=None, preprocess=True, encode_labels=<function _SupervisedExperiment.predict_model.<locals>.encode_labels at 0x000002612E3989D0>)
2023-09-14 11:18:36,687:INFO:Checking exceptions
2023-09-14 11:18:36,687:INFO:Preloading libraries
2023-09-14 11:18:36,690:INFO:Set up data.
2023-09-14 11:18:36,699:INFO:Set up index.
2023-09-14 11:20:23,207:INFO:Initializing predict_model()
2023-09-14 11:20:23,207:INFO:predict_model(self=<pycaret.classification.oop.ClassificationExperiment object at 0x000002612D906470>, estimator=ExtraTreesClassifier(bootstrap=False, ccp_alpha=0.0, class_weight=None,
                     criterion='gini', max_depth=None, max_features='sqrt',
                     max_leaf_nodes=None, max_samples=None,
                     min_impurity_decrease=0.0, min_samples_leaf=1,
                     min_samples_split=2, min_weight_fraction_leaf=0.0,
                     n_estimators=100, n_jobs=-1, oob_score=False,
                     random_state=0, verbose=0, warm_start=False), probability_threshold=None, encoded_labels=False, raw_score=False, round=4, verbose=True, ml_usecase=None, preprocess=True, encode_labels=<function _SupervisedExperiment.predict_model.<locals>.encode_labels at 0x000002612E3989D0>)
2023-09-14 11:20:23,207:INFO:Checking exceptions
2023-09-14 11:20:23,207:INFO:Preloading libraries
2023-09-14 11:20:23,210:INFO:Set up data.
2023-09-14 11:20:23,218:INFO:Set up index.
